go/client/team_cli_rendering.go

```diff
@@ -76,44 +79,42 @@ func (c *teamMembersRenderer) outputRole(team, role string, members []keybase1.T
 	}
 }
 
-func (c *teamMembersRenderer) formatInviteName(invite keybase1.AnnotatedTeamInvite) (res string) {
-	res = string(invite.Name)
-	category, err := invite.Type.C()
-	if err == nil {
-		switch category {
-		case keybase1.TeamInviteCategory_SBS:
-			res = fmt.Sprintf("%s@%s", invite.Name, string(invite.Type.Sbs()))
-		case keybase1.TeamInviteCategory_SEITAN:
-			if res == "" {
-				res = "<token without label>"
-			}
-		}
-	}
-	return res
-}
-
 func (c *teamMembersRenderer) outputInvites(invites map[keybase1.TeamInviteID]keybase1.AnnotatedTeamInvite) {
-	for _, invite := range invites {
+	for _, annotatedInvite := range invites {
+		invite := annotatedInvite.Invite
 		category, err := invite.Type.C()
 		if err != nil {
 			category = keybase1.TeamInviteCategory_UNKNOWN
 		}
-		trailer := fmt.Sprintf("(* invited by %s, awaiting acceptance)", invite.InviterUsername)
+		trailer := fmt.Sprintf("(* invited by %s)", annotatedInvite.InviterUsername)
+		inviteIDTrailer := ""
+		if c.showInviteID {
```

Comment:
would you mind also fixing this? its setup is missing in `newTeamMembersRenderer`